### ISSUE TITLE
Global Themer Layouts or .DAT file as Themer Layout

### ISSUE THREAD

**Issue by Bobolinos** [beaverbuilder/feature-requests#78]
I have a multi site and want to change the blog layout (and other layouts) with themer globally. It would be nice to be able to change this globally in the network admin environment. Or it would be nice to load a themer layout from the theme as a .dat file for example and update that .date file now and then. 

This type of layout or themer part should not be able to be updated from within the website itself, only from network admin or by updating the .dat file. 

I understand the website needs to have the correct ACF fields etc. but those can also be in the theme as well with local sync for example. I also understand some things might break if ACF fields etc are missing for example, but this is my problem to deal with.

Maybe even local sync (like ACF has) for Themer layouts/parts would be a good idea.

**Comment by kbysfm** [comment]
This would be a killer feature!

**Comment by Bobolinos** [comment]
Yeah I think so too! Also see this request which kinda can do the same but in a different way: https://github.com/beaverbuilder/feature-requests/issues/79
